articles/azure-maps/about-azure-maps.md

```diff
@@ -22,7 +22,7 @@ Azure Maps is a collection of geospatial services and SDKs that use fresh mappin
 * Various routing options; such as point-to-point, multipoint, multipoint optimization, isochrone, electric vehicle, commercial vehicle, traffic influenced, and matrix routing.
 * Traffic flow view and incidents view, for applications that require real-time traffic information.
 * Mobility services (Preview) to request public transit information, plan routes by blending different travel modes and real-time arrivals.
```

Comment:
According to the [Microsoft Writing Style Guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=29088), the word preview should not be in parentheses. This impacts sections other than the geolocation sections where it is being removed.